packages/blunderbuss/README.md

```diff
@@ -54,6 +54,11 @@ assignee.
 
 Pull Requests will not be assigned a user while in draft mode.
 
+### Assigning to a team
+
+You can configure teams for issues or PRs by using the format `<organization>/<team-name>` (no leading `@`).
+Blunderbuss with expand the entire team roster into the pool of possible assignees.
```

Comment:
Nit: minor typo, should probably be "Blunderbuss will ..." instead of "Blunderbuss with".